Website/src/utils/api.ts

```diff
@@ -32,9 +32,20 @@ export type FullTrack = BareTrack & {
 
 export type TrackList = BareTrack[];
 
+export const POITypeIconValues = {
```

Comment:
I used this instead of an enum for the website code, as the "backwards mapping" automatically added by TypeScript would have forced me to write messier code. This is functionally equivalent in most ways to an enum, when combined with the type definition below.